compiler/resolution/implementForallIntents.cpp

```diff
@@ -1910,27 +2151,18 @@ static void createShadowVarsNew(Expr* refNode, SymbolMap& uses,
         // And anyway, only 'ref' intent makes sense here.
         pruneit = true;
 
-        USR_WARN(refNode,
-                 "sync, single, or atomic var '%s' currently can be passed into the forall loop by 'ref' intent only - %s is ignored",
-                 ovar->name,
-                 tiMarker ? intentDescrString(tiMarker->intent) : "default intent");
-
-      } else if (isRecordWrappedType(ovar->type) &&
-                 !(tiIntent & INTENT_FLAG_REF)) {
-        // Threading through the leader for non-ref intents
-        // may not work correctly for arrays/domains, so we avoid it.
-        tiIntent = (tiIntent & INTENT_FLAG_CONST) ?
-          INTENT_CONST_REF : INTENT_REF;
-        USR_WARN(refNode, "Arrays, domains, and distributions currently can be passed into the forall loop by 'const', 'ref' or 'const ref' intent only. '%s' will be passed by %s.", ovar->name, intentDescrString(tiIntent));
+        USR_WARN(refNode, "sync, single, or atomic var '%s' currently can be passed into the forall loop by 'ref' intent only - %s is ignored", ovar->name, intentDescrString(tiIntent));
```

Comment:
I'm confused - it looks like this code is in a if isAtomicType but the error message is about sync/single/atomic. Am I missing something? Or could/should we update the error message to be only about atomic.